client/src/types.ts

```diff
@@ -12,10 +12,17 @@ export interface ChatData {
 }
 
 export interface ChatMsg {
+  // we should change name, color, and isUser to just mentorId
+  name: string;
```

Comment:
I thought we were getting rid of `name`, `color` and possibly `isUser` for this interface. Do we still need them?

Reply:
Yes, we said so yesterday but the chat.tsx component was no longer working because these two attributes were missing. Do you want me to remove name, color, and isUser and try to fix the chat.tsx

Reply:
no we can leave it for now in that case